### DISCUSSION TITLE
Does 4bit support offloading yet?

### DISCUSSION THREAD

**Discussion by ye7iaserag** [oobabooga/text-generation-webui#370] [Q&A] [answered]
Want to try llama 30b 4bit and there is no way to load that on 3080ti without offloading, I noticed we just got 8bit offloading support, but does that also work with 4bit quantized models?

**Comment by BetaDoggo** [comment, marked as answer]
It's now possible as of [`7618f3f`](https://github.com/oobabooga/text-generation-webui/commit/7618f3fe8c0d1fad6fdc6f7d99f0346b74c8e535) using the `--gptq-pre-layer <number of layers>` argument.

**Reply by ye7iaserag** [reply]
Yep responded to it here https://github.com/oobabooga/text-generation-webui/issues/177#issuecomment-1476960189
Thanks a lot
